### PR TITLE
i18n(es): update routing

### PR DESCRIPTION
**Pull request by kevinzunigacuellar**
#### What kind of changes does this PR include?

- Updated content
- Translated content

#### Description

- Update `routing.md` #1173

### CODE REVIEW

File: src/pages/es/core-concepts/routing.md [hunk 1]
```diff
@@ -21,9 +21,17 @@ src/pages/posts/1.md         -> mysite.com/posts/1
 ```
 
 :::tip
-Las páginas estáticas se crean automáticamente al colocar archivos en la carpeta `/src/pages/` sin ninguna configuración adicional.
+¡No hay una "configuración de enrutamiento" separada para mantener en un proyecto Astro! Cuando agrega un archivo al directorio `/src/pages`, se crea automáticamente una nueva ruta. En compilaciones estáticas, puedes personalizar el formato de salida del archivo usando la opción configuración [`build.format`](/es/reference/configuration-reference/#buildformat)
 :::
 
+## Navegando entre páginas
+
+Astro utiliza HTML estándar [`<a>`](https://developer.mozilla.org/es/docs/Web/HTML/Element/a) para navegar entre rutas. No se proporciona ningún componente `<Link>` específico en Astro.
```
Comment (threkk) [inline src/pages/es/core-concepts/routing.md:29]: Word order :)
```suggestion
Astro utiliza estándar de HTML [`<a>`](https://developer.mozilla.org/es/docs/Web/HTML/Element/a) para navegar entre rutas. No se proporciona ningún componente `<Link>` específico en Astro.
```

**Review by threkk — COMMENTED** [review]
Hello! Thanks for the contribution! I have added a couple of comments of things that sound weird. It is understandable, but a native speaker would find them odd.

**Comment by kevinzunigacuellar** [comment]
Good go to! ✅

**Comment by Yan-Thomas** [comment]
Thanks @kevinzunigacuellar, @threkk! Amazing job! 💜

**Pull request merged**